dotnet/sidebars.js

```diff
@@ -22,12 +22,13 @@ module.exports = {
         { type: 'doc', id: 'actionability' },
         { type: 'doc', id: 'api-testing' },
         { type: 'doc', id: 'test-assertions' },
+        { type: 'doc', id: 'input' },
         { type: 'doc', id: 'auth' },
         { type: 'doc', id: 'browsers' },
         { type: 'doc', id: 'browser-contexts' },
```

Comment:
IIRC, this is now `Isolation` and should be moved as well.

Reply:
good catch. Fixed.